bpmn-model/src/main/java/io/zeebe/model/bpmn/validation/zeebe/StartEventValidator.java

```diff
@@ -37,9 +36,10 @@ public void validate(StartEvent element, ValidationResultCollector validationRes
       validationResultCollector.addError(0, "Start event can't have more than one type");
     } else {
       for (EventDefinition eventDef : eventDefinitions) {
-        if (!SUPPORTED_EVENT_TYPE.isAssignableFrom(eventDef.getClass())) {
+        if (!TimerEventDefinition.class.isAssignableFrom(eventDef.getClass())
```

Comment:
Since we know the reference type we're comparing against, I would use `instanceof` here instead.

```suggestion
        if (!(eventDef instanceof TimerEventDefinition || eventDef instanceof MessageEventDefinition))
```